l10n_ch_invoice_reports/models/mail_template.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+import base64
+
+from odoo import _, models
+
+
+class MailTemplate(models.Model):
+    _inherit = "mail.template"
+
+    def generate_email(self, res_ids, fields=None):
+        # Method overridden in order remove QR/ISR payslips and use
+        # Invoice report with payslip generated by this module
+
+        result = super().generate_email(res_ids, fields)
+        if self._name == "account.move":
```

Comment:
`self._name` CANNOT be `account.move`
We don't have tests for this thing, right? :sweat_smile: 

What about using `self.model_id.model`?